### PR TITLE
Added basic documentation for topics and dashboards

### PR DESCRIPTION
**Pull request by santostiago**

### CODE REVIEW

**Comment by tiagojsag** [comment]
I don't know if this is accurate or not, please as @pablopareja or someone more familiar with this to review

File: source/index-rw.html.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ includes:
   - api/graph
   - api/areas
   - api/webshot
+  - api/topic
```
Comment (tiagojsag) [inline source/index-rw.html.md:27]: if we're making these services part of the core, then you may also need to update the other index files for gfw prep and the default one - you may want to reach out to the respective PMs on this, or check with @pablopareja

Comment (santostiago) [reply source/index-rw.html.md:27]: @pablopareja can you confirm this?

**Comment by tiagojsag** [comment]
@pablopareja @santostiago updates here?

**Review by tiagojsag — APPROVED** [review]

**Comment by tiagojsag** [comment]
Fixes https://github.com/resource-watch/doc-api/issues/21

**Pull request merged**